addons/l10n_in/static/src/js/hsn_autocomplete.js

```diff
@@ -0,0 +1,100 @@
+/** @odoo-module **/
+
+import { AutoComplete } from "@web/core/autocomplete/autocomplete";
+import { registry } from "@web/core/registry";
+import { _t } from "@web/core/l10n/translation";
+import { CharField, charField } from "@web/views/fields/char/char_field";
+
+const l10n_in_hsn_service_url = "https://services.gst.gov.in/commonservices/hsn/search/qsearch";
```

Comment:
Why does the js need the URL?  I thought we were going through a controller previously...  (the browser does not complain about crossdomain?)

Reply:
@mart-e

Reply:
Probably the government server won't complain...

Reply:
No problem on runbot at least